src/API/index.js

```diff
@@ -1,87 +1,121 @@
-import axios from 'axios';
+const axios = require('axios');
 
-function errorHandling(err) {
-  /* eslint-disable no-console */
-  let errorText = '';
-  if (err.response) {
-    console.log('Error Status', err.response.status);
-    console.log('Error Downloading', err.response.data);
-    errorText = err.response.data;
-  } else if (err.request) {
-    console.log('No response was received', err.request);
-    errorText = 'There was no response from the server.';
-  } else {
-    console.log('Unknown Error', err.message);
-    errorText = err.message;
-  }
-  return errorText;
-  /* eslint-enable no-console */
+const protocol = 'http';
+const host = process.env.ROBOKACHE_HOST || 'lvh.me';
+const port = 8080;
+
+/**
+ * URL Maker
+ * @param {string} ext extension to append to url
+ */
+const base_url = `${protocol}://${host}:${port}/api/`;
+
+function handleAxiosError(error) {
+	let errorResponse;
+	if (error.response) {
+		errorResponse = error.response.data;
+		errorResponse.status = "error";
```

Comment:
You're setting errorResponse to a string in the line right above this.

Reply:
`error.response.data` is a JSON object. I'm setting the status property of that object so that it's easy to manage error handling in the UI with `response.status == 'error'`.